src/filters/FiltersReducer.js

```diff
@@ -22,9 +22,6 @@ let initialState = {
 const filters = (state = initialState, action) => {
   switch (action.type) {
 
-  case types.DATA_CONFIG_REQUEST:
-    return {...state, loadingFilters: true};
```

Comment:
`An alternative approach is to use the object spread syntax proposed for the next versions of JavaScript which lets you use the spread (...) operator to copy enumerable properties from one object to another in a more succinct way.`

You heard it here folks